shared/wallets/transaction-details/index.tsx

```diff
@@ -322,6 +344,29 @@ const TransactionDetails = (props: NotLoadingProps) => {
       </Kb.Box2>
       <Kb.Divider />
       <Kb.Box2 direction="vertical" gap="small" fullWidth={true} style={styles.container}>
+        {isPathPayment && (
+          <Kb.Box2 direction="vertical" gap="xtiny" fullWidth={true}>
+            <Kb.Text type="BodySmallSemibold">Conversion rate:</Kb.Text>
+            <Kb.Box2 direction="horizontal" gap="small" fullWidth={true}>
+              <ConvertedCurrencyLabel
+                amount={1}
+                assetCode={props.sourceAsset}
+                issuerDescription={
+                  // If we don't have a sourceAsset, the source is native Lumens
+                  props.sourceAsset === '' ? 'Stellar Lumens' : props.sourceIssuer
```

Comment:
When transferring XLM to an asset or an asset to XLM, it made the most sense to me to just set the "issuer" field to be `Stellar Lumens`. Open to other suggestions, though.